packages/react-native-reanimated/src/ConfigHelper.ts

```diff
@@ -52,6 +54,11 @@ export function addWhitelistedUIProps(props: Record<string, boolean>): void {
   }
 }
 
+export function configureLogger(config: LoggerConfig) {
+  updateLoggerConfig(config);
+  executeOnUIRuntimeSync(updateLoggerConfig)(config);
+}
```

Comment:
What about other runtimes? Maybe we should instead do `configureLoggerOnJS`, `configureLoggerOnUI` for now?

Reply:
You mean we would want to have a separate configuration for both runtimes? I think it will be a bit confusing and, by default, the user would expect to change the configuration for all runtimes (e.g. the user doesn't have to know if the error/warning comes from the UI or JS runtime and it will be hard to predict if the change made to the config will work as intended).

I can agree that current configuration options are a bit limited and we might think of adding more options. Having just a `level` and `strict` fields don't provide enough configuration options.

Reply:
Ok, it makes sense now. Yeah, this might be problematic. I didn't have other runtimes in my mind.